src/components/integrated/OakQuizCheckBox/OakQuizCheckBox.tsx

```diff
@@ -163,14 +186,25 @@ export const OakQuizCheckBox = (props: OakQuizCheckBoxProps) => {
     </OakFlex>
   );
 
+  const backgroundColor: OakCombinedColorToken =
+    disabled && !isFeedback ? "bg-neutral-stronger" : "bg-primary";
+
+  const feedbackBgColor: OakCombinedColorToken = isCorrect
+    ? "bg-correct"
+    : "bg-incorrect";
+
+  const feedbackBorderColor = isCorrect ? "border-success" : "border-error";
+
   const inputCheckbox = (
     <StyledFlexBox
       $pa="inner-padding-l"
       $borderRadius={"border-radius-m2"}
       $borderColor={"border-primary"}
-      $background={disabled ? "bg-neutral-stronger" : "bg-primary"}
+      $background={backgroundColor}
       $flexGrow={1}
       onClick={handleContainerClick}
+      overlayBorderColor={isFeedback ? feedbackBorderColor : undefined}
```

Comment:
🤔 If i'm following this correctly setting `isCorrect` won't do anything unless `isFeedback` is also true. The need for both could be helped with docs but its a little confusing.

Could this be simplified by replacing both booleans with a union?

Giving us something like:

```jsx
<>
  <OakQuizCheckbox feedback="correct" />
  <OakQuizCheckbox feedback="incorrect" />
</>
```

What do you think?

Reply:
yes that's right. I can see that's a bit cleaner but it's perhaps less so in the wider context. Each question has a feedback state boolean which can be easily passed on as a prop (see below). On the other hand I can now pass the feedback directly on instead. Radio buttons will also need a neutral feedback state. ... hmmm okay I'll do it :)

```
const isCorrect = questionState.feedback?.[index] === "correct";
        return (
          <OakQuizCheckBox
            key={`${questionUid}-answer${index}`}
            id={`${questionUid}-answer${index}`}
            value={answerText ? answerText.text : ""}
            isFeedback={isFeedbackMode}
            isCorrect={isCorrect}
            innerRef={innerRefs.current[index]}
            onChange={handleOnChange}
          />
        );
   ```